DQM/GEM/interface/GEMDQMBase.h

```diff
@@ -26,6 +26,10 @@
 #include "Geometry/GEMGeometry/interface/GEMGeometry.h"
 #include "Geometry/Records/interface/MuonGeometryRecord.h"
 
+// 0: region, 1: station, 2: later, 3: module, 4: chamber or iEta
```

Comment:
All other keys and maps are defined in GEMValidationUtils.h: please do the same also for the 5Ids

Reply:
Hi @perrotta, 

I'm afraid that `GEMUtils` in `GEMValidationUtils.h` is wildly used and it would cause a lot of change on other parts, especially validation parts. In my opinion, it should be taken over to another PR.

Reply:
Why? `ME5IdsKey` and `MEMap5Ids` do not enter anywhere else than in this new code, adding their definitions in GEMValidationUtils.h cannot affect any other code in CMSSW

Reply:
Sorry for my late...

I was concerned that more parts like `getSuffixName` and `getSuffixTitle` functions in `GEMValidationUtils.h` should be extended with the new keys. But I've realized that it seems not necessary for now, and just moving the definitions should be okay now. Thanks for your careful advice!